ui/src/views/People/Add.vue

```diff
@@ -17,6 +17,8 @@
       <DynamicForm
         :fields="fields"
         :key="dynamicFormKey"
+        :isActivePractitioner="isActivePractitionerDefault"
```

Comment:
These don't need to be passed in. Update the fields object to set the default values.